src/libsync/logger.cpp

```diff
@@ -37,6 +37,7 @@ namespace {
 
 constexpr int CrashLogSize = 20;
 constexpr auto MaxLogLinesCount = 50000;
+constexpr auto MaxLogLinesBeforeFlush = 5;
```

Comment:
feel free to suggest a better value
I am unsure what would be the best balance between impacting performance and not loosing too much logs in case of a crash

Reply:
changed to `10`